### ISSUE TITLE
Accessibility: Carousel button must have accessible text

### ISSUE THREAD

**Issue by Viale-Stephane** [ng-bootstrap/ng-bootstrap#4417]
### Bug description:

Hello, when running an accessibility analysis tool (axe chrome extension) on the carousel demo page an issue occur.
The buttons to navigate in the list of pictures don't have any text (either with a visually-hidden label or an aria-label) but are labelled by the aria-labelledby attribute: https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/carousel/carousel.ts#L84
and the aria-labelledby attribute links to a div item. from [documentation](https://www.w3.org/TR/using-aria/#label-support), div and span when combined with aria-labelledby attribute should have a role.

Adding a role would fix the issue but the text speech would read everything inside the aria-labelledby element (the div). Which means it would read what is inside the ngTemplateOutlet see [here](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/carousel/carousel.ts#L97). (you can test it with a text reader on the demo page of the carousel)

Last issue with those buttons comes from when they are hidden. They are accessible through keyboard (check axe scan OR use a screen reader and tab on the second demo carousel), while next and previous button are not accessible. It comes from the fact you've handled differently the way to hide those two types of buttons. for the next/previous button you remove them from the dom when "[showNavigationArrows](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/carousel/carousel.ts#L112)" is false.
While for the list buttons they are just visually hidden see [here](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/carousel/carousel.ts#L83).
And visually-hidden elements are accessible elements.

### Step that reproduces the issue:

- access the demo page: https://ng-bootstrap.github.io/#/components/carousel/examples
- run axe accessibility analysis
- issue is reported
![image](https://user-images.githubusercontent.com/34646444/200781977-6761109f-a62b-4e12-be22-f6674a1f2eb6.png)

additional screen with list button visually hidden as shown in the demo page:
![image](https://user-images.githubusercontent.com/34646444/200792648-181b8b3e-ff83-44eb-9a45-1d09a9a2f8ad.png)

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: (your latest demo page, 5.2.0)

ng-bootstrap: (your latest demo page, 13.1.0)

Bootstrap: (your latest demo page, 14.1.0)

Thanks !

**Comment by bastienmoulia** [comment]
Maybe it should be made in the native bootstrap component first: https://github.com/twbs/bootstrap/issues/31908
